templates/vue.webapp/src/App.vue

```diff
@@ -0,0 +1,79 @@
+<template>
+  <div id="app">
+    <h2 class="title">project-name</h2>
+    <div class="logo">
+      <img :src="logoSVG" height="150px" alt="logo" />
+    </div>
+    <div>
+      This project is generated with
+      <b>
+        <a href="https://github.com/shpota/goxygen">goxygen</a>
+      </b>.
+      <p />The following list of technologies comes from
+      a REST API call to the Go-based back end. Find
+      and change the corresponding code
+      in
+      <code>webapp/src/tech/Tech.js</code> and
+      <code>server/web/app.go</code>.
+      <Tech />
+    </div>
+  </div>
+</template>
+
+<script>
+import Tech from './components/Tech.vue'
+
+export default {
+  name: 'App',
+  components: {
+    Tech
+  },
+  data(){
```

Comment:
A missing space. Could you please apply formatting.

Reply:
Thank you. I will refactor it